server/txSender/txSender_test.go

```diff
@@ -127,7 +127,8 @@ func TestTxSender_SendTxs(t *testing.T) {
 		},
 	}
 	args.TxNonceHandler = &testscommon.TxNonceSenderHandlerMock{
-		ApplyNonceAndGasPriceCalled: func(ctx context.Context, address core.AddressHandler, tx *transaction.FrontendTransaction) error {
+		ApplyNonceAndGasPriceCalled: func(ctx context.Context, txs ...*transaction.FrontendTransaction) error {
+			require.Len(t, txs, 1) // we update transactions one at a time
```

Comment:
Maybe...
Should we refactor to create all txs and then send them all in bulk?

Reply:
I think I tried this and didn't work as expected, not sure if it was my bad or the sdk didn't work properly when spammed.
I'd say we keep it one at the time for now.

Reply:
Can we add a task to check if we send them in bulk?

It's better logic anyway... have them all formatted + send them all;
Instead, right now, we can format 2/3 correct + send them, but fail for the third one...

I think it's better to use the ""all or nothing" approach.